joomla/joomla-cms
# Category manager - banner item count

This patch creates in "**Category Manager: Banners**" 4 new columns with the number of banners per category. It contains a helper file for com_banners that is used in com_categories to calculate the number of banner items (published, unpublished, archived, and trashed) in each category.
## Test instructions:

The **Category Manager: Banners** (Components > Banners > Categories) does not display the number of banners in each category.

![screen shot 2015-05-13 at 14 24 31](http://issues.joomla.org/uploads/1/2f2c591a294686240a72aa874d4afbba.png)

Create some new banners that are unpublished, trashed & archived (tip: name the unpublished banner "Unpublished Banner" to test if the calculations are correct).

![screen shot 2015-05-13 at 14 24 31](http://issues.joomla.org/uploads/1/d40fe4207200ab0b1e66f76c933fa9b6.png)

**Note** This patch works only **in combination with** #6916. Install both #6916 and this patch (the order of installation does not matter, you just need both).
After installing the patch, The **Category Manager: Banners** (Components > Banners > Categories) should displays 4 new columns.

![screen shot 2015-05-13 at 14 24 31](http://issues.joomla.org/uploads/1/ac3b5f74c6e3af32d3c47ac645a45e0b.png)
### Please also check the other administrator template **Hathor**

In Extensions > Template Manager > Select Hathor
Before the patch:

![screen shot 2015-05-13 at 14 24 31](http://issues.joomla.org/uploads/1/1fcb0a762ef2e9f9e9f9c102a966d157.png)
After the patch:
![screen shot 2015-05-13 at 14 24 31](http://issues.joomla.org/uploads/1/3840d02bcd4bd2d4e5695b5fe234db7e.png)
